src/oas2/transformers/__tests__/responses.test.ts

```diff
@@ -94,4 +95,75 @@ describe('responses', () => {
       expect(responses[0].contents![0].examples).toHaveLength(2);
     });
   });
+
+  describe('schema examples', () => {
+    it('given schema examples should translate to responses', () => {
```

Comment:
```suggestion
    it('given a response with a schema with an example', () => {
```

Also, this should be broke in multiple describe and a single `it` assertion, but this is not blocking.